git/git.go

```diff
@@ -203,6 +203,18 @@ func ReadBranchConfig(branch string) (cfg BranchConfig) {
 	return
 }
 
+func DeleteLocalBranch(branch string) error {
+	configCmd := GitCommand("branch", "-d", branch)
+	_, err := run.PrepareCmd(configCmd).Output()
```

Comment:
Nit: to only check error status without looking into the command output, you can use `Run()` instead of `Output()`

Reply:
Thanks!